Packs/SAP_IAM/Integrations/SAPIAM/SAPIAM.py

```diff
@@ -0,0 +1,267 @@
+    details = res.get('error', {}).get('detail')
+    return f'{message}: {details}'
+
+
+'''COMMAND FUNCTIONS'''
+
+
+def test_module(client: Client, params: dict):
+    """ Tests connectivity with the client. """
+
+    res = client.test(params)
+    if res.status_code == 200:
+        return_results('ok')
+    else:
+        return_results(res)
+
+
+def get_mapping_fields(client: Client) -> GetMappingFieldsResponse:
+    """ Creates and returns a GetMappingFieldsResponse object of the user schema in the application
+
+    :param client: (Client) The integration Client object that implements a get_app_fields() method
+    :return: (GetMappingFieldsResponse) An object that represents the user schema
+    """
+    app_fields = client.get_app_fields()
+    incident_type_scheme = SchemeTypeMapping(type_name=IAMUserProfile.DEFAULT_INCIDENT_TYPE)
+
+    for field, description in app_fields.items():
+        incident_type_scheme.add_field(field, description)
+
+    return GetMappingFieldsResponse([incident_type_scheme])
+
+
+def main():
+    user_profile = None
+    params = demisto.params()
+    base_url = params.get('url').strip('/')
+    deactivate_uri = params.get('deactivate_uri')
+    identifier = params.get('credentials', {}).get('identifier')
+    password = params.get('credentials', {}).get('password')
+    mapper_in = params.get('mapper_in')
+    mapper_out = params.get('mapper_out')
+    verify_certificate = not params.get('insecure', False)
+    proxy = params.get('proxy', False)
+    command = demisto.command()
+    args = demisto.args()
+    user_id = args.get('user-profile').get('id')
+    email = args.get('user-profile').get('email')
+    user_name = args.get('user-profile').get('username')
+
+    is_create_enabled = params.get("create_user_enabled")
+    is_enable_enabled = params.get("enable_user_enabled")
+    is_disable_enabled = params.get("disable_user_enabled")
+    is_update_enabled = params.get("update_user_enabled")
+    create_if_not_exists = params.get("create_if_not_exists")
+
+    iam_command = IAMCommand(is_create_enabled, is_enable_enabled, is_disable_enabled, is_update_enabled,
+                             create_if_not_exists, mapper_in, mapper_out, 'username')
+
+    headers = {
+        'Content-Type': 'application/json',
+        'Accept': 'application/json',
+    }
+
+    client = Client(
+        base_url=base_url,
+        verify=verify_certificate,
+        proxy=proxy,
+        ok_codes=(200, 201),
+        headers=headers,
+        auth=(identifier, password),
+        deactivate_uri=deactivate_uri,
+        user_id=user_id,
+        email=email,
+        user_name=user_name
+    )
+
+    demisto.debug(f'Command being called is {command}')
+
+    '''CRUD commands'''
+
+    if command == 'iam-get-user':
+        user_profile = iam_command.get_user(client, args)
+
+    elif command == 'iam-disable-user':
+        user_profile = iam_command.disable_user(client, args)
```

Comment:
what about `create` and `update`?
if they are not supported, remove them from the yml file and from the command examples file.

also, you can remove the relevant parameters from the configurations.

@dantavori any objections?